index.html

```diff
@@ -3,11 +3,16 @@
 
 <head>
     <title>Project UFEM</title>
+    <link rel="stylesheet" type="text/css" href="./css/styles.css" />
 </head>
 
 <body>
-    <div class="banner">
-        sth is gonna be there
+    <div class="cookie-banner">
+        <p> We use cookies to improve your experience. Please read our <a
+                href="https://policies.google.com/technologies/cookies?hl=en-US">cookie policy</a> for more information
+            about how we use
+            cookies.</p>
```

Comment:
This looks like a content that can change easy (e.g.: more paragraphs).
If that happen, we might have problem with positioning that content with `.cookie-banner__close`.

Can we wrap this part of _Cookie-Banner_ Module with additional wrapper?
```suggestion
<div class="cookie-banner__content">
  <p>
    We use cookies to improve your experience. Please read our
    <a href="https://policies.google.com/technologies/cookies?hl=en-US">cookie policy</a>
    for more information about how we use cookies.
  </p>
</div>
```